foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/sessions/cdi/InjectionManagerImpl.java

```diff
@@ -47,7 +47,11 @@
     public InjectionManagerImpl(Object beanManagerInstance) throws NamingException {
         if (beanManagerInstance == null) {
             Context context = new InitialContext();
-            beanManagerInstance = context.lookup("java:comp/BeanManager");
+            try {
+				beanManagerInstance = context.lookup("java:comp/BeanManager");
```

Comment:
this looks to be using tabs instead of spaces

Reply:
> how is this different from #734 ?

Thanks for tour comments. I did the #734 with github online and it does not work so i create this new one following the wiki  https://github.com/eclipse-ee4j/eclipselink/wiki/Creating-a-Pull-Request
I can close 734, I am a litte lost actually to achieve this pull request

Reply:
i will using space instead of tabs

Reply:
Why Travis CI is rejected again my build ?

Reply:
You can see it at line 51. There should be 16 space characters instead of current 4 tabs.
It seems, that Your IDE doesn't have expected code formatter.  See e.g.
https://github.com/eclipse-ee4j/eclipselink/wiki/Setting-up-Eclipse